contrib/cirrus/packer/prior_fedora_base-setup.sh

```diff
@@ -1 +0,0 @@
-fedora_base-setup.sh
```

Comment:
What happened here? File mixup? @cevich

Reply:
Whoa, wha?  This file should be a symlink to `fedora_base-setup.sh`